katdal/chunkstore.py

```diff
@@ -286,6 +286,13 @@ def split(cls, name, maxsplit=-1):
         """Split chunk name into components based on supported separator."""
         return name.split(cls.NAME_SEP, maxsplit)
 
+    @classmethod
+    def chunk_id_str(cls, slices):
+        """Chunk identifier in string form (e.g. '00012_01024_00000')."""
+        index = [s.start for s in slices]
+        return '_'.join(["{:0{width}d}".format(i, width=cls.NAME_INDEX_WIDTH)
+                         for i in index])
```

Comment:
Doesn't really seem necessary to create the `index` intermediate list - it could all be done in one line comprehension.

Reply:
Sure, that would be my first instinct too, but I'm trying to be gentler these days to other readers of the code :-)

In this particular case, I can buy your arguments - reverting to old habits...